section/storage.html

```diff
@@ -1,23 +1,167 @@
+                Grant? grant;
+                
+                // Payment Rate Information
+                PaymentRateInfo? rateInfo;
+            };
+            </pre>
+            
+            <dl>
+                <dt><dfn>wallet</dfn></dt>
+                <dd>Contains the user's wallet address information and server endpoints.<br><span class="note">Set when a wallet address is successfully retrieved and validated.</span></dd>
+                
+                <dt><dfn>keys</dfn></dt>
+                <dd>Contains the cryptographic key pair used for HTTP message signatures.<br><span class="note">Generated during initial setup and persisted for subsequent requests.</span></dd>
+                
+                <dt><dfn>grant</dfn></dt>
+                <dd>Contains the active grant tokens and management URLs.<br><span class="note">Set when a grant request is successfully completed and approved.</span></dd>
+                
+                <dt><dfn>rateInfo</dfn></dt>
+                <dd>Contains payment rate configuration for the current monetization session.<br><span class="note">This information is configured by the user.</span></dd>
+            </dl>
+        </section>
+        
+        <section data-dfn-for="AuthKeys">
+            <h4>Authentication Keys</h4>
+            
+            <p>The {{AuthKeys}} dictionary contains the cryptographic key pair used for HTTP message signatures in Open Payments requests.</p>
+            
+            <pre class="idl">
+            dictionary AuthKeys {
+                required DOMString keyId;
+                required DOMString privateKey;
+                required DOMString publicKey;
+            };
+            </pre>
+            
+            <dl>
+                <dt><dfn>keyId</dfn></dt>
+                <dd>The key identifier used in HTTP signature headers to reference this specific key pair.</dd>
+                
+                <dt><dfn>privateKey</dfn></dt>
+                <dd>The private key for signing HTTP messages. Must be stored securely and encrypted.</dd>
+                
+                <dt><dfn>publicKey</dfn></dt>
+                <dd>The corresponding public key that can be shared with servers for signature verification.</dd>
+            </dl>
+        </section>
+        
+        <section data-dfn-for="PaymentRateInfo">
+            <h4>Payment Rate Information</h4>
+            
+            <p>The {{PaymentRateInfo}} dictionary contains payment rate configuration that controls the frequency and amount of Web Monetization payments.</p>
+            
+            <pre class="idl">
+            dictionary PaymentRateInfo {
+                DOMString rateOfPay;
+                DOMString maxRateOfPay;
+            };
+            </pre>
+            
+            <dl>
+                <dt><dfn>rateOfPay</dfn></dt>
+                <dd>The current rate of payment in the wallet's asset units per time interval (e.g., "1000" for 1000 units per second).</dd>
```

Comment:
I think we should define this conversion/operation as an algorithm somewhere.

We also have in "Outgoing payment and quote grant request" section, which follows same logic on converting a human-friendly float value to _scaledValue_ as per `assetScale` of wallet:
> Let _scaledValue_ be <code>Math.floor(parseFloat(_budget_)) * Math.pow(10, _assetScale_))</code>.